public/app/features/provisioning/setup_warnings.ts

```diff
@@ -0,0 +1,25 @@
+import { AppEvents, FeatureToggles } from '@grafana/data';
+import { config, getAppEvents } from '@grafana/runtime';
+
+let sent = 0;
+const requiredFeatureToggles: Array<keyof FeatureToggles> = [
+  'kubernetesFolders',
+  'kubernetesDashboards',
+  'unifiedStorageSearch',
+];
+
+export function checkSetup() {
```

Comment:
@Clarity-89 -- is there a better way to do this?  useEffect?

Reply:
umm, instead of popup, should it take a more central space in the screen? It's pretty critical for the feature.

Reply:
Yeah, probably should be a custom hook that calls the check logic in `useEffect`.